pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -75,6 +75,13 @@ func newReconciler(mgr manager.Manager) reconcile.Reconciler {
 	return &ReconcileInfinispan{client: mgr.GetClient(), scheme: mgr.GetScheme()}
 }
 
+// NewFakeReconciler creates a new fake Reconciler for unit testing
```

Comment:
Is this just for testing? It would be great to keep it separated from implementation code. A separated file would be good, if possible

Reply:
to move it to the separate file, it would be need to move kubernetes and cluster vars inside ReconcileInfinispan struct and have all fields to be exported

Reply:
Is this need also if moved in the same package?

Reply:
Thanks, @rigazilla , just requires RestClient rest.Interface to be changed and exported